### DISCUSSION TITLE
Change which config items get ENabled by default in Home Assistant?

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js-ui#3909] [General]
When I add a new device, when it shows up in home assistant a number of configuration parameters I want are DISabled. I then have to manually go ENable them every time.

Is there any way to change that behavior and have more of them ENabled by default?

**Comment by kpine** [comment]
That's a question for Home Assistant, not here. Unless you are using MQTT.

And no, you cannot, at least for the Z-Wave integration.

Please clarify how you are using it.

**Reply by Botched1** [reply]
Thanks for the answer!

> Please clarify how you are using it.

For instance on the GE/Jasco Motion Dimmer/Switch (26931 / 26933) I regularly change/use the Operation Mode, Timeout Duration, and Switch Level (for dimmers) configuration parameters in my automations. All of those config entities are DISabled by default.  So I have to turn the on when pairing a new device, or if I try a new controller as that seems to also reset which device entities are enabled/disabled back to the defaults on the HA side..

**Reply by kpine** [reply]
Yeah, but are you using the Z-Wave integration or MQTT Discovery? Whether an entity is enabled by default is decided either by a) the Z-Wave integration, or b) ZUI for MQTT discovery.

**Reply by Botched1** [reply]
Oh duh, yes I misinterpreted the question. 

I'm using the Z-Wave / home assistant integration (not mqtt), so it makes sense that how it is set up in HA is all on the home assistant side of things.

Maybe I should go back and look at the mqtt route again. I'm very familiar with mqtt in general, as I use it for a number of things including at work.

**Reply by kpine** [reply]
Ok, HA disables it by default because, I would say, most people don't need to automate the config settings, and those that do usually only need a handful. On the extreme, enabling 30 entities by default for no reason would be a worse experience for most people. I don't think they would change that.

MQTT also disables these by default, so switching doesn't help. Although you might be able to request it to be an option, I don't know. The integration is recommended for use with HA though, but you have the option.
